packages/terra-dialog/package.json

```diff
@@ -32,8 +32,9 @@
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "terra-base": "^2.0.0",
-    "terra-clinical-action-header": "^1.6.0",
-    "terra-content-container": "^1.15.0"
+    "terra-button": "^1.14.0",
```

Comment:
Let's set these to `^2.0.1` for button, content-container, and icon.

Reply:
https://github.com/cerner/terra-core/pull/1250/commits/26e7f056a00ff51fe00bffc670a7d53f9db26c6d